independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/QuarkusClassLoader.java

```diff
@@ -156,6 +172,8 @@ public void reset(Map<String, byte[]> resources,
             } catch (ClassNotFoundException ignored) {
                 //ignore
             }
+            //this is another huge hack to deal with JDK services
+            //as we have to return null from getParent() (see
```

Comment:
This comment seems out of place, no?

Reply:
oops, I was about to start an even hackier approach here then I realised I could just use the platform CL as the parent..